build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e
+dotnet --info
+dotnet restore
+
+for path in src/**/*.csproj; do
+    dotnet build -f netstandard1.3 -c Release ${path}
+    dotnet build -f netcoreapp1.1 -c Release ${path}
```

Comment:
The target frameworks seems off here... There's no `netcoreapp1.1` *or* `netstandard1.3` projects under `src`.

Reply:
Copy-pasta; thanks 👍